### ISSUE TITLE
Ugdb crashes 

### ISSUE THREAD

**Issue by shirshak55** [ftilde/ugdb#13]
Here are the errors

```
Oh no! ugdb crashed!
Consider filing an issue including the log file located in /tmp and the following backtrace at https://github.com/ftilde/ugdb:

panicked at 'spawn gdb: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/libcore/result.rs:1165:5
stack backtrace:
   0: ugdb::run::{{closure}}
   1: std::panicking::rust_panic_with_hook
             at src/libstd/panicking.rs:468
   2: std::panicking::continue_panic_fmt
             at src/libstd/panicking.rs:373
   3: rust_begin_unwind
             at src/libstd/panicking.rs:302
   4: core::panicking::panic_fmt
             at src/libcore/panicking.rs:139
   5: core::result::unwrap_failed
             at src/libcore/result.rs:1165
   6: ugdb::run
   7: ugdb::main
   8: std::rt::lang_start::{{closure}}
   9: std::rt::lang_start_internal::{{closure}}
             at src/libstd/rt.rs:48
      std::panicking::try::do_call
             at src/libstd/panicking.rs:287
  10: __rust_maybe_catch_panic
             at src/libpanic_unwind/lib.rs:78
  11: std::panicking::try
             at src/libstd/panicking.rs:265
      std::panic::catch_unwind
             at src/libstd/panic.rs:396
      std::rt::lang_start_internal
             at src/libstd/rt.rs:47
  12: main
  13: __libc_start_main
  14: _start
```

Host: 
```
Linux ubuntu 4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:28 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by ftilde** [comment]
No need to close this early! The issue here was apparently that the gdb binary could not be spawned. This is obviously an error that can happen on accident during 'normal' usage and should not result in a crash. Should be fixed in 0b79a15f8159fe10e75e366b51f8e1b436a0de9a.

**Comment by shirshak55** [comment]
@ftilde sorry i felt too dumb honestly. Actually i came here after reading amos tut i ran `cargo install ugdb` and expected `ugdb` to run automatically but had forgotten to use ```sudo apt install gdb```. 

By the way thanks  for giving us this awesome tui tool:)
